eox_nelp/openedx_filters/xapi/filters.py

```diff
@@ -198,3 +199,45 @@ def run_filter(self, transformer, result):  # pylint: disable=arguments-differ,
         return {
             "result": result
         }
+
+
+class XApiContextFilter(PipelineStep):
+    """This filter is designed to update the context value for any event transformer.
+
+    How to set:
+        OPEN_EDX_FILTERS_CONFIG = {
+            "event_routing_backends.processors.xapi.transformer.xapi_transformer.get_context": {
+                "pipeline": ["eox_nelp.openedx_filters.xapi.filters.XApiContextFilter"],
+                "fail_silently": False,
+            },
+        }
+    """
+
+    def run_filter(self, transformer, result):  # pylint: disable=arguments-differ, unused-argument
+        """Modifies statement context for any event
+
+        Arguments:
+            transformer <XApiTransformer>: Transformer instance.
+            result <Context>: Context related to an event.
```

Comment:
Here could be an example of the shape of the XAPI_EXTRA_CONTEXT. I mean the key is the attribute name and the value could be a dict with definitions.